models.py

```diff
@@ -0,0 +1,100 @@
+import exceptions
+import random
+import settings
+
+
+
+class Player:
+    """ Main class for game mechanics"""
+
+
+    def __init__(self, name):
+        self.name = name
+        self.lives = settings.LIVES
+        self.score = 0
+
+
+    @staticmethod
+    def fight(attack, defense):
+        """Main game rules"""
+        if attack == defense:
+            return 0
+        elif (attack == settings.WARRIOR and defense == settings.ROGUE) or \
+            (attack == settings.ROGUE and defense == settings.WIZARD) or \
+            (attack == settings.WIZARD and defense == settings.WARRIOR):
+            return 1
+        else:
+            return -1
+
+    def attack(self, enemy):
+        """Attack mechanics for player"""
+        print(self.name + ' attack now:')
+        while True:
+            print('Choose your hero: 1 for Warrior, 2 for Wizard, 3 for Rogue')
+            your_attack = int(input())
+            if your_attack in [settings.WARRIOR, settings.WIZARD, settings.ROGUE]:
+                break
+
+        enemy_attack = Enemy.select_attack()
```

Comment:
you have the enemy object right here and can select_attack method from the object.
Not critical but more clear.